crates/genesis/Cargo.toml

```diff
@@ -14,6 +14,8 @@ exclude.workspace = true
 [dependencies]
 alloy-primitives.workspace = true
 alloy-rpc-types.workspace = true
+alloy-signer.workspace = true
+k256.workspace = true
```

Comment:
we should re-export this from alloy-signer actually

Reply:
k256 u mean right?